packages/core/src/config/load.ts

```diff
@@ -2,35 +2,43 @@ import * as fs from 'fs';
 
 import { RedoclyClient } from '../redocly';
 import { loadYaml } from '../utils';
-import { Config, RawConfig } from './config';
+import {
+  Config,
+  RawConfig,
+  Region,
+  resolveRedoclyDomain,
+} from './config';
 
 import { defaultPlugin } from './builtIn';
 
-export async function loadConfig(configPath?: string, customExtends?: string[]): Promise<Config> {
-  if (configPath === undefined) {
-    configPath = findConfig();
-  }
-  let rawConfig: RawConfig = {};
+export async function loadConfig({
```

Comment:
Breaking change:

Using in redocly-portal there: https://github.com/Redocly/redocly-portal/blob/main/engine/plugins/gatsby-source-redocly-filesystem/oas/openapi-cli-bundle.js#L33